.github/workflows/ci.yml

```diff
@@ -41,11 +45,14 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '25.3'
+
       - name: Use paulo-ferraz-oliveira/setup-elp
         id: setup-elp
         uses: ./
         with:
-          input-1: input-1
+          elp-version: '2024-06-07'
 
-      - name: Output input-1 (from output-1)
-        run: echo "${{ steps.setup-elp.outputs.output-1 }}"
+      - run: elp version
```

Comment:
If this works, it means installation for ELP for the arch + platform + OTP (derived from the `erl` execution) went Ok